### PR TITLE
Condense JIT/Quantization triage into one workflow

### PR DESCRIPTION
**Pull request by samestep**
The `.github/workflows/{jit,quantization}_triage.yml` workflows are nearly identical, so this PR consolidates them into a single GitHub Actions workflow to reduce code duplication. It also renames the workflow so it starts with a capital letter, so that it will show up alongside all our other GitHub Actions workflows on [the HUD](https://hud.pytorch.org/build2/pytorch-master).

### CODE REVIEW

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -7,8 +7,19 @@ on:
 jobs:
   welcome:
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: JIT
+            column: Need triage
```
Comment (walterddr) [inline .github/workflows/triage.yml:15]:
```suggestion
            column: Need Triage
```

Comment (samestep) [reply .github/workflows/triage.yml:15]: @walterddr no, the entire point of having `column` in the matrix is that the column names are different; compare:
- https://github.com/pytorch/pytorch/projects/13
- https://github.com/pytorch/pytorch/projects/17

**Review by walterddr — APPROVED** [review]
lgtm. although i dunno what's the value of the `column` variable added. seems the workflow name itself is "Triage"?

**Comment by samestep** [comment]
> lgtm. although i dunno what's the value of the `column` variable added. seems the workflow name itself is "Triage"?

@walterddr see https://github.com/pytorch/pytorch/pull/61130#discussion_r662460795

File: .github/workflows/triage.yml (hunk 1 above)
Comment (walterddr) [reply .github/workflows/triage.yml:15]: Ahh. LOL.. dunno why we have 2 different naming convention then. but fine.

Comment (samestep) [reply .github/workflows/triage.yml:15]: @walterddr yeah I wish they were the same 😅 as you can see in the first commit of this PR, I initially thought they were, but then I had to add a second commit when I realized that they are not

**Review by zhouzhuojie — APPROVED** [review]
🎉

File: .github/workflows/triage.yml [hunk 2]
```diff
@@ -18,25 +29,26 @@ jobs:
...
-              core.debug("Issue " + issue.data.title + " does not have JIT label");
+            if (!hasLabel) {
+              core.debug(`Issue ${issue.data.title} does not have ${kName} label`);
               return;
             }
 
             // Get project column ID.
-            const kProjectName = "JIT Triage";
-            const kColumnName = "Need triage";
+            const kProjectName = `${kName} Triage`;
```
Comment (janeyx99) [inline .github/workflows/triage.yml:50]: Just curious, why is the convention to have k's starting every variable name?

**Review by janeyx99 — APPROVED** [review]

File: .github/workflows/triage.yml (hunk 2 above)
Comment (samestep) [reply .github/workflows/triage.yml:50]: @janeyx99 Some coding styles prefix `const`ant names with `k`. Before this PR, all these variables actually are literal constant strings, but I guess after this PR, their values are determined by the matrix so they're not really constants; perhaps I should have removed the `k` prefix.

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**